src/c-writer.cc

```diff
@@ -2246,9 +2246,12 @@ void CWriter::Write(const SimdLaneOpExpr& expr) {
 void CWriter::Write(const SimdShuffleOpExpr& expr) {
   Type result_type = expr.opcode.GetResultType();
   Write(StackVar(1, result_type), " = ", expr.opcode.GetName(), "(",
-        StackVar(1), " ", StackVar(0), ", lane Imm: $0x%08x %08x %08x %08x",
-        expr.val.v[0], expr.val.v[1], expr.val.v[2], expr.val.v[3], ")",
-        Newline());
+        StackVar(1), " ", StackVar(0), ", lane Imm: %llu %llu %llu %llu %llu %llu %llu %llu %llu %llu %llu %llu %llu %llu %llu %llu",
```

Comment:
I'm not sure why "lane" was used here before, but I've left it as is. 

It might be worth it to change this to `i5x16 Imm:` instead or similar.